python_apps/pypo/pypo/pypofetch.py

```diff
@@ -264,7 +264,7 @@ def update_liquidsoap_stream_format(self, stream_format):
             command = ('vars.stream_metadata_type %s\n' % stream_format).encode('utf-8')
             self.logger.info(command)
             tn.write(command)
-            tn.write('exit\n')
+            tn.write('exit\n'.encode('ascii'))
```

Comment:
Rather encode as utf-8, even though it won't affect the output, it maintains consistency with the other internationalisation support

Reply:
Ok. There are a few other areas in pypo that need to be updated as well for python3. I'm having trouble finding them so I'll keep working on this.

Reply:
cool. I'm happy to merge this once you've found them

Reply:
Just fixed them. Can happily reinstall LibreTime then get it to run after copying and pasting those start service commands. Also the stream restarts properly when I change details in stream settings. So it seems to be working well now 🤞🏽